plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationSshj.java

```diff
@@ -298,6 +303,24 @@ private int setPortForwarding(
         return ss.getLocalPort();
     }
 
+    private static class FilterLoggerFactory implements LoggerFactory {
+        private static final Set<String> FILTERED_OUT_CLASSES = Set.of("net.schmizz.sshj.common.StreamCopier");
```

Comment:
Not sure that it is enough.
I see some other sshj logs like this:
```
2023-07-27 17:31:18.051 - Disconnected - BY_APPLICATION
2023-07-27 17:31:18.051 - In pipe from < ChannelInputStream for Channel #1 > to java.net.Socket$SocketOutputStream@45a19449
net.schmizz.sshj.transport.TransportException: [BY_APPLICATION] Disconnected
	at net.schmizz.sshj.transport.TransportImpl.disconnect(TransportImpl.java:396)
	at net.schmizz.sshj.transport.TransportImpl.disconnect(TransportImpl.java:387)
	at net.schmizz.sshj.transport.TransportImpl.disconnect(TransportImpl.java:382)
	at net.schmizz.sshj.SSHClient.disconnect(SSHClient.java:439)
	at org.jkiss.dbeaver.model.net.ssh.SSHImplementationSshj$1.run(SSHImplementationSshj.java:192)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-07-27 17:31:18.056 - <<copyDone>> woke to: net.schmizz.sshj.transport.TransportException: [BY_APPLICATION] Disconnected
```

Reply:
This warning comes from the same class, so it's enough